ml_api/requirements.txt

```diff
@@ -1,9 +1,7 @@
 ipdb
-opencv-python>=3.4
```

Comment:
Any reason why we want to move opencv related module to a separate requirement file?

Reply:
pip doesn't have opencv modules for ARM in their repo. The ubuntu-ports python-opencv worked as an alternative. I moved them so only x86 based machines try to download them with pip. I figured it was easier than finding an alternative pip package that supported x86 and ARM.